File: Amigo.Tenant.UI.Web/src/app/model/moduleDTO.ts
import  {ActionDtoModule} from './actiondto';
 export class ModuleDto {

     public code: string;
     public name: string;
     public url: string;
     public parentModuleCode: string;
     public sortOrder: number;
     public onlyParents: boolean;
     public actions : ActionDtoModule[]; 


 }

File: Amigo.Tenant.Application.DTOs/Responses/Security/ModuleDTO.cs
using System;


namespace Amigo.Tenant.Application.DTOs.Responses.Security
{
    public class ModuleDTO
    {

        public string Code
        {
            get; set;
        }

        public string Name
        {
            get; set;
        }

        public string Url
        {
            get; set;
        }

        public string ParentModuleCode
        {
            get; set;
        }

        public string ParentModuleName
        {
            get; set;
        }


        public int SortOrder
        {
            get; set;
        }

        public bool? OnlyParents
        {
            get; set;
        }
    }
}
